package.json

```diff
@@ -93,7 +93,7 @@
 		"build-client-if-prod": "node -e \"process.env.CALYPSO_ENV === 'production' && process.exit(1)\" || yarn run build-client-both && yarn run build-languages-if-enabled",
 		"build-client-if-desktop": "node -e \"(process.env.CALYPSO_ENV === 'desktop' || process.env.CALYPSO_ENV === 'desktop-development') && process.exit(1)\" || yarn run build-client-fallback",
 		"build-client-stats": "NODE_ENV=production CALYPSO_ENV=production EMIT_STATS=true yarn run build-client-evergreen",
-		"build-packages": "{ [ \"$SKIP_TSC\" == \"true\" ] || tsc --build packages/tsconfig.json; } && lerna run prepare --stream",
+		"build-packages": "{ [ \"$SKIP_TSC\" = \"true\" ] || tsc --build packages/tsconfig.json; } && lerna run prepare --stream",
```

Comment:
Drive-by fix: `==` is a bash-only thing, it fails when running as part of a Docker build process because it uses POSIX `sh`